scripts/us_ca_chhs/healthcare_facility_bed_counts/preprocess_csv_helper.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import numpy as np
+
+# This function is to return a map{county_name: geoId} given CA county name.
+def generate_dcid_for_county(counties):
+    county_to_geoID = {}
+    sorted_counties = np.sort(counties)
+    # see browser.datacommons.org/kg?dcid=geoId/06 for convention for geoId to county.
+    counties_id = [i for i in range(1, len(counties) * 2 + 1, 2)]
+    for i in range(len(sorted_counties)):
+        county_to_geoID[sorted_counties[i]] = perserve_length(counties_id[i])
+    return county_to_geoID
+
+
+# A helper function to generate geoId for given county name.
```

Comment:
Seems to me this function returns the geoId of a county given its fips code. Maybe make the name reflect that? Also I thought all counties have exactly 3 digits, so you could probABLY USE zfill.